sdk/nodejs/package.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "@pulumi/mysql",
```

```suggestion
    "name": "@unobravo/pulumi-mysql",
```

Credo che forse facendo la modifica del commento precedente e ribuildando magari si cambia. (Perchè c'è anche da altre parti)